src/tools/gophish_cleaner.py

```diff
@@ -160,17 +159,16 @@ def main() -> None:
         )
     except ValueError:
         logging.critical(
-            '"{}"is not a valid logging level.  Possible values are debug, info, warning, and error.'.format(
-                log_level
-            )
+            '"%s"is not a valid logging level.  Possible values are debug, info, warning, and error.',
```

Comment:
Do you need an extra space here?
```suggestion
            '"%s" is not a valid logging level.  Possible values are debug, info, warning, and error.',
```

Reply:
see 825962be74cfc038bdf689b711d0b91860d5c0b5